joomla/joomla-cms
# [5.1] Convert mod_login to service provider

### Summary of Changes

Converts the administrator login module to service provider.

### Testing Instructions

1. You should see module output in administrator login page

![mod_login](https://github.com/joomla/joomla-cms/assets/59166858/a78a02d1-88c6-4d6f-8344-f86a89d2496f)

### Actual result BEFORE applying this Pull Request

Module works as expected

### Expected result AFTER applying this Pull Request

Module works same way as before applying pull request

### Link to documentations
Please select:
